internal/sqlsmith: TestGenerateParse failed

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3946176&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3946176&tab=artifacts#/) on master @ [3f95a4bd83cce2952a12497de82692866b4da659](https://github.com/cockroachdb/cockroach/commits/3f95a4bd83cce2952a12497de82692866b4da659):


```
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:907 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc006206f88, {0x6284f80, 0xc00304f630})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc001876580, 0x2, 0x3}, 0x925d2c0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc001a24280, 0x3)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc001a24280, 0x6347560)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc00304f6d0, 0x4573cc0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x10000c004302308, 0x925d2c0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x0, {0x6285720, 0xc00304f6d0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*ParenSelect).doc(0xc0062072a8, 0x6285200)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:479 +0x2a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4b861d8, {0x6285200, 0xc0021925e0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Subquery).doc(0xc003690f90, 0x6285f00)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:653 +0x6d
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x45e054, {0x6285f00, 0xc003690f90})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*AliasedTableExpr).doc(0xc00304f720, 0x6283bc0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:665 +0x59
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc006207448, {0x6283bc0, 0xc00304f720})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc007c00c60, 0x1, 0x1}, 0x925d2c0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc001a24320, 0x1)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc001a24320, 0x6347560)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc00304f770, 0x4573cc0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x45ad0d, 0x925d2c0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc006207710, {0x6285720, 0xc00304f770})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x925d2c0, {0x6285720, 0xc00304f770})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x6289ee0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc007f9c340)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc007f9c340, 0x5275f28)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (0.61s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3945166&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3945166&tab=artifacts#/) on master @ [3f95a4bd83cce2952a12497de82692866b4da659](https://github.com/cockroachdb/cockroach/commits/3f95a4bd83cce2952a12497de82692866b4da659):


```
SELECT
	'1971-01-02 19:45:05.000158':::TIMESTAMP AS col_80
FROM
	(VALUES ('3000-01-01 00:00:00':::TIMESTAMP)) AS tab_51 (col_79)
WHERE
	true;

STMT: 33
COMMIT TRANSACTION;

STMT: 34
SELECT
	tab_54.col_84 AS col_86
FROM
	(
		VALUES
			('13:46:54.413062':::TIME),
			('18:04:44.44623':::TIME),
			('06:40:59.470831':::TIME),
			('12:00:19.886629':::TIME),
			('10:36:35.124852':::TIME),
			('24:00:00':::TIME)
	)
		AS tab_52 (col_81),
	(VALUES (0.2726545035839081:::FLOAT8, NULL), (0.3813754916191101:::FLOAT8, (-1.272452203849319):::FLOAT8))
		AS tab_53 (col_82, col_83),
	(
		VALUES
			(4701:::INT8, e'\t':::STRING),
			(
				st_zmflag(
					'01050000000900000001020000000500000070BF4510F092C6C1295725227C8AF1C10CE33C73CF80E94164858CB469DAF8C1F0205D10F66B0042B41D6E1BA73DFF41BC4563FD33C0E1417AABEE67CCE90142D7F457515CF1F9C13CAB2440EA44E54101020000000400000086F11CB6EB36EEC11A2EB909EE5AE3C17229A450E5A4F74172D907D841D0F0C1B2120CC7A0A0F141D076B2A4370ED2412A6058225998F541987977742AC7DE4101020000000400000088F9BF6AA623DEC1C022723EFE4FB9C110785162FDA8F941AD890DE93031F0C1E89D8FF49118FE41BAB3E9CAAB35EFC18AF7C507D30EF5C142406306EBEFF04101020000000400000048409BD5585DF041202BCACC4DE7C6C16EDCF1A27068F241164A1C4D37BCF44140485459CABECA416A3A09C5F719FA41D4208F64873FDAC13EF7CCB5A77FF241010200000003000000AE04C40AD2BCF44178C3FB704D7FFFC142963A621795F241F6DB01847400F34180E30BDFD0E1E941709C12B778BAF7410102000000050000005297C6E99C3AFFC179F1DBC7944C00C2B854CA36A567D54158022395F90000C21CD21BE39A39FC41EBD21F7FF96402C2EAE2069E7979F3413648D47E6256EBC12C453C8DE223E74128BC6E239536DB41010200000003000000B427DA432E4DE941482729966D19F5C1A08849FA143EF04128D9035D47A7D14118D4D46EBF74E9415CF82D63155AEB410102000000030000009A06279D37D0E4C169E05877D8F9FDC1408680329AFDCA41FA4D4DAF067FE8C12A94523672DFF5416F1917F4356901C2010200000006000000A86E6BE50003E941E186AB887F16F4C1AC4809899382F8417F6ACD030D61F5C1481963221F86ED415CC135367B6FE9C102DD718BF49AF541FC42A807A573EA411C24AE51FBB4E741D4129F4287A5FA418A747D69C71FE4C15C34499653F6FA41':::GEOMETRY::GEOMETRY
				)::INT2,
				'}':::STRING
			),
			(NULL, 'R':::STRING),
			((-32618):::INT8, NULL),
			(4372:::INT8, '``':::STRING)
	)
		AS tab_54 (col_84, col_85)
WHERE
	true
LIMIT
	66:::INT8;

    sqlsmith_test.go:190: ALTER DATABASE defaultdb: at or near "EOF": syntax error
W211218 10:43:01.758527 13753 kv/kvserver/intentresolver/intent_resolver.go:790  [-] 696  failed to gc transaction record: could not GC completed transaction anchored at /Table/SystemConfigSpan/Start: node unavailable; try another peer
W211218 10:43:01.758817 12883 sql/sqlliveness/slinstance/slinstance.go:222  [n1] 697  exiting heartbeat loop
W211218 10:43:01.758820 13061 jobs/registry.go:851  [n1] 698  canceling all adopted jobs due to stopper quiescing
--- FAIL: TestGenerateParse (0.28s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3945169&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3945169&tab=artifacts#/) on master @ [3f95a4bd83cce2952a12497de82692866b4da659](https://github.com/cockroachdb/cockroach/commits/3f95a4bd83cce2952a12497de82692866b4da659):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*ValuesClause).docTable(0xc001bb56c8, 0xaf8ff88)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:1066 +0x105
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc001149c20, 0xc00295ba40)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0x157
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0xc0040b8cb8, 0xdf0baa0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x3b
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x61d9c00, {0xaeccec0, 0xc001149c20})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*ParenSelect).doc(0xc0028826d0, 0xaecc9a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:479 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x6db1e21, {0xaecc9a0, 0xc0028826d0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Subquery).doc(0xc002d36690, 0xaecd6a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:653 +0x9b
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc0040b8ef0, {0xaecd6a0, 0xc002d36690})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*AliasedTableExpr).doc(0xc001149c70, 0xaecb360)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:665 +0x65
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xaece040, {0xaecb360, 0xc001149c70})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc000d72980, 0x1, 0x1}, 0xdf0baa0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0x13c
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc0017475e0, 0x1)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0xcd
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc0017475e0, 0xc001158900)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x2dc
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).doc(0xc00096a380, 0xdf0baa0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:559 +0x3b
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xdf0baa0, {0xaeccee0, 0xc0017475e0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*With).docRow(0xc00107ebc0, 0xc002882728)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:640 +0x2a5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc001149e00, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:546 +0x85
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x4c2225, 0xdf0baa0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x3b
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xa, {0xaeccec0, 0xc001149e00})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0xdf0baa0, {0xaeccec0, 0xc001149e00})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x45
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0xaed1680)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x7b
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc000f5ba00)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x9db
        testing.tRunner(0xc000f5ba00, 0x749d578)
        	/usr/local/go/src/testing/testing.go:1259 +0x230
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1306 +0x727
--- FAIL: TestGenerateParse (3.47s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-race -parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3951499&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3951499&tab=artifacts#/) on master @ [4cbe795c7f8c36c007c3ca1250367cb181f5ca0c](https://github.com/cockroachdb/cockroach/commits/4cbe795c7f8c36c007c3ca1250367cb181f5ca0c):


```
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x4530280, 0x8e4c730})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc000d98240, {0x6313560, 0xc00007e048})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x4530280, 0x8e4c730})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).formatNodeOrHideConstants(0x8e5a860, {0x0, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/hide_constants.go:46 +0x192
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).FormatNode(0xc0029d6b60, {0x0, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:444 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.AsStringWithFlags({0x0, 0x0}, 0x48040, {0x0, 0x10, 0xc00060c400})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:582 +0x69
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).docAsString(0x203000, {0x0, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:160 +0x35
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4b77712, {0x0, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:155 +0x65
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*OnJoinCond).doc(0xc004701310, 0x9262380)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:932 +0x85
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x45e3a7, {0x7f0552df44c8, 0xc001ceadb0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*JoinTableExpr).doc(0xc000f0d720, 0x62881c0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:907 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc004701448, {0x62881c0, 0xc000f0d720})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc001ceadc0, 0x1, 0x3}, 0x9262380)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc00180d5e0, 0x3)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc00180d5e0, 0x634a880)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc000f0d770, 0x4576380)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x45ad0d, 0x9262380)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc004701710, {0x6288960, 0xc000f0d770})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x9262380, {0x6288960, 0xc000f0d770})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x628d120)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc00193e820)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc00193e820, 0x5278db0)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (0.37s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3950457&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3950457&tab=artifacts#/) on master @ [4cbe795c7f8c36c007c3ca1250367cb181f5ca0c](https://github.com/cockroachdb/cockroach/commits/4cbe795c7f8c36c007c3ca1250367cb181f5ca0c):


```
        	/usr/local/go/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x44ddd40, 0x8bed720})
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc0019c0360, {0x62c2ee0, 0xc000074108})
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x44ddd40, 0x8bed720})
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).formatNodeOrHideConstants(0x8bfb720, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/hide_constants.go:46 +0x192
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).FormatNode(0xc000cca8c0, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:444 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.AsStringWithFlags({0x0, 0x0}, 0x48040, {0x0, 0x10, 0xc000500400})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:582 +0x69
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).docAsString(0x203002, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:160 +0x35
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4b24432, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:155 +0x65
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*OnJoinCond).doc(0xc0076cb310, 0x90029e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:932 +0x85
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x451c07, {0x7f01af243420, 0xc0085b9af0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*JoinTableExpr).doc(0xc004af46e0, 0x6237a20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:907 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc0076cb448, {0x6237a20, 0xc004af46e0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc0085b9b00, 0x1, 0x4}, 0x90029e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc004e1f720, 0x4)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc004e1f720, 0x62fa200)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc004af4730, 0x4523dc0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x44e56d, 0x90029e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc0076cb710, {0x62381c0, 0xc004af4730})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x90029e0, {0x62381c0, 0xc004af4730})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x623c980)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc0055fc1a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc0055fc1a0, 0x52256d8)
        	/usr/local/go/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (0.27s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3950461&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3950461&tab=artifacts#/) on master @ [4cbe795c7f8c36c007c3ca1250367cb181f5ca0c](https://github.com/cockroachdb/cockroach/commits/4cbe795c7f8c36c007c3ca1250367cb181f5ca0c):


```
        	/usr/local/go/src/runtime/debug/stack.go:24 +0x72
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x94
        panic({0x6757f40, 0xdaf7780})
        	/usr/local/go/src/runtime/panic.go:1047 +0x266
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc0021a0240, {0xaf5dac0, 0xc0001c2008})
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x49a
        panic({0x6757f40, 0xdaf7780})
        	/usr/local/go/src/runtime/panic.go:1047 +0x266
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).formatNodeOrHideConstants(0xc0001e3490, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/hide_constants.go:46 +0x1d9
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).FormatNode(0xc0001e3490, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:444 +0x12d
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.AsStringWithFlags({0x0, 0x0}, 0x48040, {0x0, 0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:582 +0x8e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).docAsString(0xc003b786f0, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:160 +0x45
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x6da4cb0, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:155 +0x91
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*OnJoinCond).doc(0xc003b78560, 0xdf12b20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:932 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc001025c40, {0x7f4f107cd280, 0xc003b78560})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*JoinTableExpr).doc(0xc002b83a90, 0xaed1960)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:907 +0x151
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xaed3280, {0xaed1960, 0xc002b83a90})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc003b78590, 0x1, 0x2}, 0xdf12b20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0x13c
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc00431d900, 0x2)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0xcd
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc00431d900, 0xaf95068)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x2dc
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc002b83ae0, 0x0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0x157
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x4c2225, 0xdf12b20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x3b
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xa, {0xaed2100, 0xc002b83ae0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0xdf12b20, {0xaed2100, 0xc002b83ae0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x45
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0xaed68c0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x7b
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc0019fc1a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x9db
        testing.tRunner(0xc0019fc1a0, 0x74a1ee0)
        	/usr/local/go/src/testing/testing.go:1259 +0x230
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1306 +0x727
--- FAIL: TestGenerateParse (2.15s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-race -parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3956365&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3956365&tab=artifacts#/) on master @ [7bd8974eca566ca7625b6621decc5ae52d4fd38e](https://github.com/cockroachdb/cockroach/commits/7bd8974eca566ca7625b6621decc5ae52d4fd38e):


```
			('00:00:00':::TIME),
			('00:08:00':::TIME),
			('00:04:00':::TIME)
	)
		AS tab_214 (col_343)
	INNER JOIN (VALUES (NULL), (NULL), (NULL), (NULL)) AS tab_215 (col_344) ON (NOT NULL),
	(
		VALUES
			((e'\\x49':::BYTES::BYTES || e'\\x45':::BYTES::BYTES)::BYTES),
			(e'\\x42':::BYTES),
			(e'\\x48':::BYTES),
			(e'\\x44':::BYTES),
			(e'\\x41':::BYTES)
	)
		AS tab_216 (col_345)
	INNER JOIN (
			VALUES
				(NULL, NULL, 'B':::STRING),
				(
					NULL,
					(
						(
							((NULL::OID[] || NULL::OID[])::OID[]::OID[] || NULL::OID[])::OID[]::OID[] || NULL::OID[]
						)::OID[]::OID[]
						|| (
								4:::OID::OID
								|| (
										NULL::OID[]
										|| ((NULL::OID[] || 0:::OID::OID)::OID[]::OID[] || 6:::OID::OID)::OID[]::OID[]
									)::OID[]::OID[]
							)::OID[]::OID[]
					)::OID[],
					NULL
				),
				(NULL, NULL, 'J':::STRING),
				(NULL, NULL, 'B':::STRING),
				(NULL, NULL, CASE WHEN true THEN 'D':::STRING ELSE 'E':::STRING END),
				(NULL, NULL, 'I':::STRING)
		)
			AS tab_217 (col_346, col_347, col_348) ON false
WHERE
	false
ORDER BY
	tab_216.col_345 ASC, tab_216.col_345, tab_214.col_343;

STMT: 72
ROLLBACK TRANSACTION;

    sqlsmith_test.go:190: ALTER DATABASE defaultdb: at or near "EOF": syntax error
--- FAIL: TestGenerateParse (0.43s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3956367&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3956367&tab=artifacts#/) on master @ [7bd8974eca566ca7625b6621decc5ae52d4fd38e](https://github.com/cockroachdb/cockroach/commits/7bd8974eca566ca7625b6621decc5ae52d4fd38e):


```
	false;

STMT: 90
COMMIT TRANSACTION;

STMT: 91
DELETE FROM defaultdb.public.table1 AS tab_80 WHERE (((NOT false) = true) AND NULL);

STMT: 92
INSERT
INTO
	defaultdb.public.table1 AS tab_81 (col1_0, col1_1)
VALUES
	(3094383642:::OID, '1974-04-07 19:46:02.000729':::TIMESTAMP);

STMT: 93
SELECT
	tab_82.crdb_internal_mvcc_timestamp AS col_67, tab_82.tableoid AS col_68
FROM
	defaultdb.public.table1@[0] AS tab_82
WHERE
	true
ORDER BY
	tab_82.crdb_internal_mvcc_timestamp;

STMT: 94
INSERT
INTO
	defaultdb.public.table1 AS tab_83 (col1_0, col1_1)
VALUES
	(325253999:::OID, '1987-03-12 11:02:56.000908':::TIMESTAMP);

STMT: 95
SELECT
	tab_84.col1_0 AS col_69,
	tab_84.crdb_internal_mvcc_timestamp AS col_70,
	tab_84.col1_1 AS col_71,
	tab_84.col1_0 AS col_72
FROM
	defaultdb.public.table1@[0] AS tab_84
WHERE
	NULL
ORDER BY
	tab_84.crdb_internal_mvcc_timestamp, tab_84.col1_1;

STMT: 96
COMMIT TRANSACTION;

    sqlsmith_test.go:190: ALTER DATABASE defaultdb: at or near "EOF": syntax error
--- FAIL: TestGenerateParse (1.22s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3955435&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3955435&tab=artifacts#/) on master @ [93c9a3bc0e234c23df45aee62f6f76eb3b5ab723](https://github.com/cockroachdb/cockroach/commits/93c9a3bc0e234c23df45aee62f6f76eb3b5ab723):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:907 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc006574f88, {0x6237ba0, 0xc0018adb30})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc007262cf0, 0x1, 0x2}, 0x90029e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc004502e60, 0x2)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc004502e60, 0x62fa380)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc0018adb80, 0x4523de0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x10000c002fec637, 0x90029e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x0, {0x6238340, 0xc0018adb80})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*ParenSelect).doc(0xc0065752a8, 0x6237e20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:479 +0x2a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4b35b59, {0x6237e20, 0xc000631a30})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Subquery).doc(0xc0007eba70, 0x6238b20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:653 +0x6d
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4518b4, {0x6238b20, 0xc0007eba70})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*AliasedTableExpr).doc(0xc0018adbd0, 0x62367e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:665 +0x59
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc006575448, {0x62367e0, 0xc0018adbd0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc007262dc0, 0x1, 0x1}, 0x90029e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc004502f00, 0x1)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc004502f00, 0x62fa380)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc0018adc20, 0x4523de0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x44e56d, 0x90029e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc006575710, {0x6238340, 0xc0018adc20})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x90029e0, {0x6238340, 0xc0018adc20})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x623cb00)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc005100340)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc005100340, 0x52257b8)
        	/usr/local/go/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (0.27s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3966640&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3966640&tab=artifacts#/) on master @ [2bf850847554519dc139289615e1a061b6a251de](https://github.com/cockroachdb/cockroach/commits/2bf850847554519dc139289615e1a061b6a251de):


```

STMT: 94
COMMIT TRANSACTION;

STMT: 95
WITH
	with_59 (col_586, col_587)
		AS (
			SELECT
				*
			FROM
				(
					VALUES
						('aed3:4b4d:45b0:6a6f:9958:a4e2:dbda:1643/40':::INET, NULL),
						('151.86.13.92/12':::INET, '1992-09-16':::DATE),
						('48.104.96.202/4':::INET, '1983-03-02':::DATE),
						(
							'9695:20ee:4aaf:3465:2be8:2dd9:8676:307b/53':::INET,
							(
								COALESCE('1973-12-24':::DATE, '1974-09-30':::DATE)::DATE
								+ 596970572357534542:::INT8::INT8
							)::DATE
						),
						((NULL::INET + unordered_unique_rowid()::INT8::INT8)::INET, '1994-05-30':::DATE)
				)
					AS tab_385 (col_586, col_587)
		)
SELECT
	cte_ref_13.col_587 AS col_589, tab_386.col_588 AS col_590
FROM
	with_59 AS cte_ref_13, (VALUES (NULL)) AS tab_386 (col_588)
WHERE
	(false OR true)
ORDER BY
	tab_386.col_588 DESC, tab_386.col_588 ASC, tab_386.col_588 DESC;

STMT: 96
WITH
	with_60 (col_591) AS (SELECT * FROM (VALUES (NULL), (2667825065:::OID), (NULL)) AS tab_387 (col_591))
SELECT
	cte_ref_14.col_591 AS col_592
FROM
	with_60 AS cte_ref_14
ORDER BY
	cte_ref_14.col_591 DESC
LIMIT
	68:::INT8;

    sqlsmith_test.go:190: ALTER DATABASE defaultdb: at or near "EOF": syntax error
--- FAIL: TestGenerateParse (0.40s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3966641&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3966641&tab=artifacts#/) on master @ [2bf850847554519dc139289615e1a061b6a251de](https://github.com/cockroachdb/cockroach/commits/2bf850847554519dc139289615e1a061b6a251de):


```
																	FROM
																		with_44 AS cte_ref_16,
																		(
																			VALUES
																				(6:::OID),
																				(9:::OID),
																				(7:::OID),
																				(7:::OID),
																				(6:::OID),
																				(9:::OID)
																		)
																			AS tab_157 (col_291)
																	WHERE
																		(NOT false)
																	GROUP BY
																		cte_ref_16.col_289
																	HAVING
																		bool_and(NULL::BOOL)::BOOL
																	LIMIT
																		63:::INT8
																)
														)
												)
											),
											(false),
											(true),
											(true),
											(true)
									)
										AS tab_158 (col_293)
								LIMIT
									1:::INT8
							)
						),
						('D':::STRING)
				)
					AS tab_159 (col_295)
		),
	with_46 (col_296) AS (SELECT * FROM (VALUES (NULL), (NULL), (NULL)) AS tab_160 (col_296)),
	with_47 (col_297) AS (SELECT * FROM (VALUES (NULL), (NULL), (NULL), (NULL), (NULL)) AS tab_161 (col_297))
SELECT
	NULL AS col_301, tab_162.col_298 AS col_302, tab_163.col_300 AS col_303, cte_ref_17.col_297 AS col_304
FROM
	(VALUES ('G':::STRING), (NULL), ('F':::STRING), ('C':::STRING)) AS tab_162 (col_298),
	(VALUES (NULL, set_bit(e'\\x44':::BYTES::BYTES, 1:::INT8::INT8, 1:::INT8::INT8)::BYTES), (NULL, NULL), (NULL, NULL))
		AS tab_163 (col_299, col_300),
	with_47 AS cte_ref_17;

    sqlsmith_test.go:190: ALTER DATABASE defaultdb: at or near "EOF": syntax error
--- FAIL: TestGenerateParse (1.05s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3976914&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3976914&tab=artifacts#/) on master @ [cadf1bf18001d00fc03f32f22d4e18b920e583ba](https://github.com/cockroachdb/cockroach/commits/cadf1bf18001d00fc03f32f22d4e18b920e583ba):


```

STMT: 41
WITH
	with_20 (col_75)
		AS (
			SELECT
				*
			FROM
				(VALUES (NULL), (2692319484:::OID), (3394861724:::OID), (0:::OID), (2900142027:::OID))
					AS tab_42 (col_75)
		),
	with_21 (col_76)
		AS (
			SELECT
				*
			FROM
				(
					VALUES
						(NULL),
						(
							st_combinebbox(
								'BOX(-2.040614423905332 -0.6299647830966498,1.1030279506703593 -0.16586993709950126)':::BOX2D::BOX2D,
								NULL::GEOMETRY
							)::BOX2D
						),
						(
							st_estimatedextent(
								e'\x7f(y(Emqc':::STRING::STRING,
								e':h\x13:_\x0bl!|':::STRING::STRING,
								'KCD':::STRING::STRING
							)::BOX2D
						)
				)
					AS tab_43 (col_76)
		)
SELECT
	'':::VOID AS col_77
LIMIT
	79:::INT8;

STMT: 42
SELECT
	tab_44.col_78 AS col_79
FROM
	(VALUES (631728887:::OID)) AS tab_44 (col_78)
ORDER BY
	tab_44.col_78 DESC, tab_44.col_78, tab_44.col_78;

    sqlsmith_test.go:190: ALTER DATABASE defaultdb: at or near "EOF": syntax error
--- FAIL: TestGenerateParse (0.30s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3976918&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3976918&tab=artifacts#/) on master @ [cadf1bf18001d00fc03f32f22d4e18b920e583ba](https://github.com/cockroachdb/cockroach/commits/cadf1bf18001d00fc03f32f22d4e18b920e583ba):


```
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:923 +0x2f8
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xa50aa20, {0xa509120, 0xc002a9c870})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc00523c430, 0x1, 0x3}, 0xd4e94a0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0x13c
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc003153680, 0x3)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0xcd
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc003153680, 0xa5cce38)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x2dc
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc002a9c8c0, 0x4c4c45)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0x157
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0xc004fbee60, 0xd4e94a0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x3b
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x60ce9e0, {0xa5098c0, 0xc002a9c8c0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*ParenSelect).doc(0xc0029b4478, 0xa5093a0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:479 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x6c8292c, {0xa5093a0, 0xc0029b4478})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Subquery).doc(0xc00649fe30, 0xa50a0a0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:653 +0x9b
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc004fbf098, {0xa50a0a0, 0xc00649fe30})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*AliasedTableExpr).doc(0xc002a9c910, 0xa507d60)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:665 +0x65
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xa50aa20, {0xa507d60, 0xc002a9c910})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc00523c4d0, 0x1, 0x1}, 0xd4e94a0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0x13c
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc003153720, 0x1)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0xcd
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc003153720, 0xa5cce38)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x2dc
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc002a9c960, 0x0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0x157
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0xc00523c4d0, 0xd4e94a0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x3b
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xa, {0xa5098c0, 0xc002a9c960})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0xd4e94a0, {0xa5098c0, 0xc002a9c960})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x45
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0xa50e060)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x7b
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc003542820)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x9db
        testing.tRunner(0xc003542820, 0x6f2c8c0)
        	GOROOT/src/testing/testing.go:1259 +0x230
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x727
--- FAIL: TestGenerateParse (2.37s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3975873&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3975873&tab=artifacts#/) on master @ [3c15deb8ff7ea864e81e13de80555d600ed56a3c](https://github.com/cockroachdb/cockroach/commits/3c15deb8ff7ea864e81e13de80555d600ed56a3c):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:907 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc00a5bef88, {0x5ca0fe0, 0xc007664fa0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc0149dd060, 0x2, 0x3}, 0x8a1d5a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc00bec2e60, 0x3)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc00bec2e60, 0x5d6a4f0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc007664ff0, 0x43d1c40)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x10000c00a5bf258, 0x8a1d5a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x0, {0x5ca1780, 0xc007664ff0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*ParenSelect).doc(0xc00a5bf2a8, 0x5ca1260)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:479 +0x2a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x49e76ff, {0x5ca1260, 0xc00072ffb0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Subquery).doc(0xc002cacb10, 0x5ca1f60)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:653 +0x6d
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4518b4, {0x5ca1f60, 0xc002cacb10})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*AliasedTableExpr).doc(0xc007665040, 0x5c9fc20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:665 +0x59
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc00a5bf448, {0x5c9fc20, 0xc007665040})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc001dc06a0, 0x1, 0x1}, 0x8a1d5a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc00bec2f00, 0x1)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc00bec2f00, 0x5d6a4f0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc007665090, 0x43d1c40)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x44e56d, 0x8a1d5a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc00a5bf710, {0x5ca1780, 0xc007665090})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x8a1d5a0, {0x5ca1780, 0xc007665090})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x5ca5f20)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc008cd1ba0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc008cd1ba0, 0x4c95718)
        	/usr/local/go/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (0.94s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=deadlock

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

```
panic.go:1038: unknown survival goal: 0
        goroutine 12325 [running]:
        runtime/debug.Stack()
        	/usr/local/go/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x45d4ce0, 0xc006e88f30})
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc004df0360, {0x5d397e0, 0xc000074118})
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x45d4ce0, 0xc006e88f30})
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SurvivalGoal).Format(0xc003d76770, 0x5c34d48)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/survival_goal.go:39 +0xae
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*AlterDatabaseSurvivalGoal).Format(0xc006e93f08, 0xc003d76770)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/alter_database.go:96 +0x78
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).formatNodeOrHideConstants(0x85a27e0, {0x5cac7c0, 0xc006e93f08})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/hide_constants.go:46 +0x19e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).FormatNode(0xc003d76770, {0x5cac7c0, 0xc006e93f08})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:444 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.AsStringWithFlags({0x5cac7c0, 0xc006e93f08}, 0x48040, {0x0, 0x3d11d01, 0xc006e93f08})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:582 +0x69
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).docAsString(0x44e56d, {0x5cac7c0, 0xc006e93f08})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:160 +0x35
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc005ced710, {0x5cac7c0, 0xc006e93f08})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:155 +0x65
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x8a28680, {0x5cac7c0, 0xc006e93f08})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x5cb29c0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc005425520)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc005425520, 0x4c9dad0)
        	/usr/local/go/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (0.91s)
```
https://teamcity.cockroachdb.com/viewLog.html?buildId=3984990&buildTypeId=Cockroach_UnitTests

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3987157&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3987157&tab=artifacts#/) on master @ [8d4953ef35bf310c72108c2790ba21971e9e7291](https://github.com/cockroachdb/cockroach/commits/8d4953ef35bf310c72108c2790ba21971e9e7291):


```
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:924 +0x21e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc0122faf88, {0x5cefc00, 0xc0072eaeb0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc0125d70a0, 0x2, 0x7}, 0x8a72600)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc0113315e0, 0x7)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc0113315e0, 0x5db91f0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc0072eaff0, 0x441ede0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x10000c0122fb258, 0x8a72600)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x0, {0x5cf03a0, 0xc0072eaff0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*ParenSelect).doc(0xc0122fb2a8, 0x5cefe80)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:479 +0x2a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4a37a64, {0x5cefe80, 0xc00c2ea8e0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Subquery).doc(0xc00ddb7770, 0x5cf0b80)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:653 +0x6d
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4541b4, {0x5cf0b80, 0xc00ddb7770})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*AliasedTableExpr).doc(0xc0072eb040, 0x5cee840)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:665 +0x59
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc0122fb448, {0x5cee840, 0xc0072eb040})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc0053a78e0, 0x1, 0x1}, 0x8a72600)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc011331680, 0x1)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc011331680, 0x5db91f0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc0072eb090, 0x441ede0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x450e6d, 0x8a72600)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc0122fb710, {0x5cf03a0, 0xc0072eb090})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x8a72600, {0x5cf03a0, 0xc0072eb090})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x5cf4b40)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc00c21d1e0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc00c21d1e0, 0x4ce6098)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (1.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss,deadlock
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3986104&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3986104&tab=artifacts#/) on master @ [964695b941eba220e7830da13e5611b7e3ed6ac6](https://github.com/cockroachdb/cockroach/commits/964695b941eba220e7830da13e5611b7e3ed6ac6):


```
STMT: 33
COMMIT TRANSACTION;

STMT: 34
SELECT
	tab_81.col_134 AS col_135
FROM
	(VALUES (NULL), (4:::OID), (4:::OID), (7:::OID), (7:::OID)) AS tab_81 (col_134)
WHERE
	false
ORDER BY
	tab_81.col_134 ASC;

    panic.go:1038: unknown survival goal: 0
        goroutine 11532 [running]:
        runtime/debug.Stack()
        	/usr/local/go/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x45d4ce0, 0xc0065a3420})
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc003526360, {0x5d39520, 0xc00013e000})
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x45d4ce0, 0xc0065a3420})
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SurvivalGoal).Format(0xc0066516c0, 0x5c34a80)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/survival_goal.go:39 +0xae
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*AlterDatabaseSurvivalGoal).Format(0xc003a15aa0, 0xc0066516c0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/alter_database.go:96 +0x78
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).formatNodeOrHideConstants(0x85a27e0, {0x5cac500, 0xc003a15aa0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/hide_constants.go:46 +0x19e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).FormatNode(0xc0066516c0, {0x5cac500, 0xc003a15aa0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:444 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.AsStringWithFlags({0x5cac500, 0xc003a15aa0}, 0x48040, {0x0, 0xc00604d601, 0xc003a15aa0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:582 +0x69
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).docAsString(0x44e56d, {0x5cac500, 0xc003a15aa0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:160 +0x35
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc00604d710, {0x5cac500, 0xc003a15aa0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:155 +0x65
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x8a28640, {0x5cac500, 0xc003a15aa0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x5cb2700)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc002005860)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc002005860, 0x4c9d9c8)
        	/usr/local/go/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (0.26s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3995500&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3995500&tab=artifacts#/) on master @ [8c1528fff5c132b971428bc7f18bf0ea5eb16799](https://github.com/cockroachdb/cockroach/commits/8c1528fff5c132b971428bc7f18bf0ea5eb16799):


```
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:907 +0x151
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xa65f800, {0xa65df00, 0xc001c8ae60})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc000f01e40, 0x1, 0x3}, 0xd6e5340)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0x13c
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc00290c960, 0x3)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0xcd
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc00290c960, 0xa7248b0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x2dc
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc001c8aeb0, 0x7f0d6bdca040)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0x157
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0xc00419ae60, 0xd6e5340)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x3b
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x61cab00, {0xa65e6a0, 0xc001c8aeb0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*ParenSelect).doc(0xc001fd4db8, 0xa65e180)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:479 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x6dbc6b2, {0xa65e180, 0xc001fd4db8})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Subquery).doc(0xc003ca4600, 0xa65ee80)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:653 +0x9b
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc00419b098, {0xa65ee80, 0xc003ca4600})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*AliasedTableExpr).doc(0xc001c8af00, 0xa65cb40)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:665 +0x65
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xa65f800, {0xa65cb40, 0xc001c8af00})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc002fb8020, 0x1, 0x1}, 0xd6e5340)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0x13c
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc00290ca00, 0x1)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0xcd
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc00290ca00, 0xa7248b0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x2dc
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc001c8af50, 0x0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0x157
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0xc002fb8020, 0xd6e5340)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x3b
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xa, {0xa65e6a0, 0xc001c8af50})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x5a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0xd6e5340, {0xa65e6a0, 0xc001c8af50})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x45
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0xa664100)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x7b
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc0011f29c0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x9db
        testing.tRunner(0xc0011f29c0, 0x706a480)
        	GOROOT/src/testing/testing.go:1259 +0x230
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x727
--- FAIL: TestGenerateParse (2.48s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3994675&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3994675&tab=artifacts#/) on master @ [8c1528fff5c132b971428bc7f18bf0ea5eb16799](https://github.com/cockroachdb/cockroach/commits/8c1528fff5c132b971428bc7f18bf0ea5eb16799):


```
        	/usr/local/go/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x4433080, 0x86fa7f0})
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc004ad6360, {0x5e15c38, 0xc000074108})
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x4433080, 0x86fa7f0})
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).formatNodeOrHideConstants(0x8708e40, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/hide_constants.go:46 +0x192
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).FormatNode(0xc0006f2b60, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:444 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.AsStringWithFlags({0x0, 0x0}, 0x48040, {0x0, 0x10, 0xc000500000})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:582 +0x69
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).docAsString(0x203000, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:160 +0x35
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4aa61f4, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:155 +0x65
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*OnJoinCond).doc(0xc0080b9310, 0x8b9a380)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:932 +0x85
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x451c07, {0x7f7e2a729590, 0xc006902650})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*JoinTableExpr).doc(0xc00446b040, 0x5d87bc0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:907 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc0080b9448, {0x5d87bc0, 0xc00446b040})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc006902660, 0x1, 0x3}, 0x8b9a380)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc005d459a0, 0x3)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc005d459a0, 0x5e4d768)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc00446b090, 0x447a600)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x44e56d, 0x8b9a380)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc0080b9710, {0x5d88360, 0xc00446b090})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x8b9a380, {0x5d88360, 0xc00446b090})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x5d8ddc0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc00634a820)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc00634a820, 0x4d69840)
        	/usr/local/go/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (0.49s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3994678&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3994678&tab=artifacts#/) on master @ [8c1528fff5c132b971428bc7f18bf0ea5eb16799](https://github.com/cockroachdb/cockroach/commits/8c1528fff5c132b971428bc7f18bf0ea5eb16799):


```
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:923 +0x1e5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc00ab22f88, {0x5d8d420, 0xc0093578b0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc006269080, 0x3, 0x2}, 0x8bab460)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc01140f220, 0x2)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc01140f220, 0x5e595c8)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc0093579f0, 0x4483e00)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x10000c00ab23258, 0x8bab460)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x0, {0x5d8dbc0, 0xc0093579f0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*ParenSelect).doc(0xc00ab232a8, 0x5d8d6a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:479 +0x2a
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4ac0345, {0x5d8d6a0, 0xc00be792d8})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Subquery).doc(0xc006b20300, 0x5d8e3a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:653 +0x6d
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4518b4, {0x5d8e3a0, 0xc006b20300})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*AliasedTableExpr).doc(0xc009357a40, 0x5d8c060)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:665 +0x59
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc00ab23448, {0x5d8c060, 0xc009357a40})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc0037d5b90, 0x1, 0x1}, 0x8bab460)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc00cdedea0, 0x1)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc00cdedea0, 0x5e595c8)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc009357a90, 0x4483e00)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x44e56d, 0x8bab460)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc00ab23710, {0x5d8dbc0, 0xc009357a90})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x8bab460, {0x5d8dbc0, 0xc009357a90})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x5d93620)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc0100ddd40)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc0100ddd40, 0x4d723d8)
        	/usr/local/go/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (1.03s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=deadlock

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4000855&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4000855&tab=artifacts#/) on master @ [f3710ae65e327d0795123029db679f8c9f3e610e](https://github.com/cockroachdb/cockroach/commits/f3710ae65e327d0795123029db679f8c9f3e610e):


```
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x4472600, 0x8731ec0})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc0038145a0, {0x5e53238, 0xc000136020})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x4472600, 0x8731ec0})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).formatNodeOrHideConstants(0x8740620, {0x0, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/hide_constants.go:46 +0x192
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).FormatNode(0xc0033e21c0, {0x0, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:444 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.AsStringWithFlags({0x0, 0x0}, 0x48040, {0x0, 0x10, 0xc0004a4000})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:582 +0x69
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).docAsString(0x203000, {0x0, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:160 +0x35
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4ae6434, {0x0, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:155 +0x65
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*OnJoinCond).doc(0xc0072bd310, 0x8bd2240)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:932 +0x85
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc0012d9840, {0x7f2f9e0b7a38, 0xc00305cce0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*JoinTableExpr).doc(0xc00266d4f0, 0x5dc52c0)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:907 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc0072bd448, {0x5dc52c0, 0xc00266d4f0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc0012d9260, 0x2, 0x3}, 0x8bd2240)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc00438b400, 0x3)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc00438b400, 0x5e8ad68)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc00266d5e0, 0x44b9c00)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x450e6d, 0x8bd2240)
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc0072bd710, {0x5dc5a60, 0xc00266d5e0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x8bd2240, {0x5dc5a60, 0xc00266d5e0})
        	github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x5dcb4c0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc000ddcea0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc000ddcea0, 0x4da9e78)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (0.50s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4000366&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4000366&tab=artifacts#/) on master @ [f3710ae65e327d0795123029db679f8c9f3e610e](https://github.com/cockroachdb/cockroach/commits/f3710ae65e327d0795123029db679f8c9f3e610e):


```
        	/usr/local/go/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x443b760, 0x870a7f0})
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc00bf3a480, {0x5e20a18, 0xc000074108})
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x389
        panic({0x443b760, 0x870a7f0})
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).formatNodeOrHideConstants(0x8718e40, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/hide_constants.go:46 +0x192
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).FormatNode(0xc00d4ba4d0, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:444 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.AsStringWithFlags({0x0, 0x0}, 0x48040, {0x0, 0x10, 0xc00ad3ec00})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:582 +0x69
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).docAsString(0x203004, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:160 +0x35
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4aadb74, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:155 +0x65
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*OnJoinCond).doc(0xc012cb9310, 0x8baa420)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:932 +0x85
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x451c07, {0x7f2b9c7d23f8, 0xc010063ad0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*JoinTableExpr).doc(0xc00d282b90, 0x5d8c3a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:907 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc012cb9448, {0x5d8c3a0, 0xc00d282b90})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc010063ae0, 0x1, 0x4}, 0x8baa420)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc00f601f40, 0x4)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc00f601f40, 0x5e58548)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc00d282be0, 0x4482de0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x44e56d, 0x8baa420)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc012cb9710, {0x5d8cb40, 0xc00d282be0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x8baa420, {0x5d8cb40, 0xc00d282be0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x5d925a0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc0086ee680)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc0086ee680, 0x4d71398)
        	/usr/local/go/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (0.63s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=deadlock

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

internal/sqlsmith.TestGenerateParse [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4000363&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4000363&tab=artifacts#/) on master @ [f3710ae65e327d0795123029db679f8c9f3e610e](https://github.com/cockroachdb/cockroach/commits/f3710ae65e327d0795123029db679f8c9f3e610e):


```
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc001230240, {0x5e15bd8, 0xc0001b2000})
        	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x4433060, 0x86fa7f0})
        	/usr/local/go/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).formatNodeOrHideConstants(0x8708e40, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/hide_constants.go:46 +0x192
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FmtCtx).FormatNode(0xc00709c9a0, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:444 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.AsStringWithFlags({0x0, 0x0}, 0x48040, {0x0, 0x10, 0x8ba3c00})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/format.go:582 +0x69
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).docAsString(0x1295654, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:160 +0x35
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x4aa61b4, {0x0, 0x0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:155 +0x65
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*OnJoinCond).doc(0xc008273220, 0x8b9a340)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:932 +0x85
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0x5d894c0, {0x7fcd846a3d98, 0xc000c88d80})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*JoinTableExpr).doc(0xc005c36190, 0x5d87b60)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:907 +0xd5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc008273398, {0x5d87b60, 0xc005c36190})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*JoinTableExpr).doc(0xc005c36230, 0x5d87b60)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:924 +0x21e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc008273448, {0x5d87b60, 0xc005c36230})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.TableExprs.doc({0xc004898620, 0x2, 0x1}, 0x8b9a340)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:272 +0xc5
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*From).docRow(0xc00184f2c0, 0x1)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:592 +0x51
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*SelectClause).docTable(0xc00184f2c0, 0x5e4d708)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:576 +0x1c7
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).docTable(0xc005c363c0, 0x447a5e0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:548 +0xdf
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*Select).doc(0x44e56d, 0x8b9a340)
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:541 +0x1e
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Doc(0xc008273710, {0x5d88300, 0xc005c363c0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:152 +0x42
        github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*PrettyCfg).Pretty(0x8b9a340, {0x5d88300, 0xc005c363c0})
        	/go/src/github.com/cockroachdb/cockroach/pkg/sql/sem/tree/pretty.go:144 +0x28
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.(*Smither).Generate(0x5d8dd60)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith.go:167 +0x6f
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.TestGenerateParse(0xc001606ea0)
        	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/sqlsmith_test.go:184 +0x786
        testing.tRunner(0xc001606ea0, 0x4d69800)
        	/usr/local/go/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	/usr/local/go/src/testing/testing.go:1306 +0x35a
--- FAIL: TestGenerateParse (0.51s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- GOFLAGS=-parallel=4
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestGenerateParse.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/internal/sqlsmith/relational.go
+++ b/pkg/internal/sqlsmith/relational.go
@@ -215,6 +215,10 @@ func makeEquiJoinExpr(s *Smither, refs colRefs, forJoin bool) (tree.TableExpr, c
 	var available [][2]tree.TypedExpr
 	for _, leftCol := range leftRefs {
 		for _, rightCol := range rightRefs {
+			// You can't compare voids.
+			if leftCol.typ.Oid() == oid.T_void || rightCol.typ.Oid() == oid.T_void {
+				continue
+			}
 			if leftCol.typ.Equivalent(rightCol.typ) {
 				available = append(available, [2]tree.TypedExpr{
 					typedParen(leftCol.item, leftCol.typ),
@@ -236,10 +240,6 @@ func makeEquiJoinExpr(s *Smither, refs colRefs, forJoin bool) (tree.TableExpr, c
 	for (cond == nil || s.coin()) && len(available) > 0 {
 		v := available[0]
 		available = available[1:]
-		// You can't compare voids.
-		if v[0].ResolvedType().Oid() == oid.T_void || v[1].ResolvedType().Oid() == oid.T_void {
-			continue
-		}
 		expr := tree.NewTypedComparisonExpr(tree.MakeComparisonOperator(tree.EQ), v[0], v[1])
 		if cond == nil {
 			cond = expr
@@ -254,7 +254,10 @@ func makeEquiJoinExpr(s *Smither, refs colRefs, forJoin bool) (tree.TableExpr, c
 		Cond:  &tree.OnJoinCond{Expr: cond},
 	}
 	joinRefs := leftRefs.extend(rightRefs...)
-	return joinExpr, joinRefs, true
+	// If we have a nil cond, then we didn't succeed in generating this join
+	// expr.
+	ok = cond != nil
+	return joinExpr, joinRefs, ok
 }
 
 func makeMergeJoinExpr(s *Smither, _ colRefs, forJoin bool) (tree.TableExpr, colRefs, bool) {
@@ -304,13 +307,19 @@ func makeMergeJoinExpr(s *Smither, _ colRefs, forJoin bool) (tree.TableExpr, col
 					if !tree.MustBeStaticallyKnownType(rightCol.Type).Equivalent(tree.MustBeStaticallyKnownType(leftCol.Type)) {
 						break
 					}
+					leftType := tree.MustBeStaticallyKnownType(leftCol.Type)
+					rightType := tree.MustBeStaticallyKnownType(rightCol.Type)
+					// You can't compare voids.
+					if leftType.Oid() == oid.T_void || rightType.Oid() == oid.T_void {
+						break
+					}
 					cols = append(cols, [2]colRef{
 						{
-							typ:  tree.MustBeStaticallyKnownType(leftCol.Type),
+							typ:  leftType,
 							item: tree.NewColumnItem(leftAliasName, leftColElem.Column),
 						},
 						{
-							typ:  tree.MustBeStaticallyKnownType(rightCol.Type),
+							typ:  rightType,
 							item: tree.NewColumnItem(rightAliasName, rightColElem.Column),
 						},
 					})
@@ -364,7 +373,10 @@ func makeMergeJoinExpr(s *Smither, _ colRefs, forJoin bool) (tree.TableExpr, col
 		},
 		Cond: &tree.OnJoinCond{Expr: cond},
 	}
-	return joinExpr, joinRefs, true
+	// If we have a nil cond, then we didn't succeed in generating this join
+	// expr.
+	ok = cond != nil
+	return joinExpr, joinRefs, ok
 }
 
 // STATEMENTS

